chapters/ch34.md

```diff
@@ -1,39 +1,33 @@
 > # 34. How to define human-level performance
 
---> _replace THIS LINE by your translation for the above line_
-
+Cách xác định hiệu quả cấp con người
 
 > Suppose you are working on a medical imaging application that automatically makes diagnoses from x-ray images. A typical person with no previous medical background besides some basic training achieves 15% error on this task. A junior doctor achieves 10% error. An experienced doctor achieves 5% error. And a small team of doctors that discuss and debate each image achieves 2% error. Which one of these error rates defines “human-level performance”?
 
---> _replace THIS LINE by your translation for the above line_
-
+Giả sử bạn đang làm việc trên một ứng dụng hình ảnh y tế tự động chẩn đoán từ hình ảnh X quang. Một người bình thường không có nên tảng y tế với một số đào tạo cơ bản có thể đạt được 15% sai số trong tác vụ này. Một bác sĩ trẻ mới ra trường đạt được 10% sai số. Một bác sĩ kinh nghiệm đạt đuợc 5% sai số. Và một nhóm nhỏ các bác sĩ thảo luận và tranh luận mỗi hình ảnh đạt được 2% sai số. Vậy cái nào trong số những sai số này xác định "hiệu quả cấp con người"?
 
 > In this case, I would use 2% as the human-level performance proxy for our optimal error rate. You can also set 2% as the desired performance level because all three reasons from the previous chapter for comparing to human-level performance apply:
 
---> _replace THIS LINE by your translation for the above line_
-
+Trong trường hợp này, tôi sẽ sử dụng 2% làm "hiệu quả cấp con người" cho tỉ lệ lỗi tối ưu của chúng ta. Bạn cũng có thể đặt 2% làm mức hiệu suất mong muốn vì nó thỏa cả ba lý do để so sánh với hiệu quả cấp con người mà chúng ta đã nói ở chương trước:
 
 > * **Ease of obtaining labeled data from human labelers.​** You can get a team of doctors to provide labels to you with a 2% error rate.
 
---> _replace THIS LINE by your translation for the above line_
-
+* **Dữ liệu được lấy dễ dàng từ người dán nhãn** Bạn có thể nhờ một nhóm bác sĩ cung cấp nhãn cho bạn với tỉ lệ lỗi 2%.
 
 > * **Error analysis can draw on human intuition.** ​By discussing images with a team of doctors, you can draw on their intuitions.
 
---> _replace THIS LINE by your translation for the above line_
+* **Phân tích lỗi có thể dựa vào trực giác của con người.** Bằng cách thảo luận về các hình ảnh với một nhóm các bác sĩ, bạn có thể lấy kết quả dựa trên được trực giác của họ.
```

Comment:
- Đoạn "lấy kết quả" có vẻ không có trong trong bản gốc (hoặc cứ cho là implicitly), và sẽ gợi ra câu hỏi "lấy kết quả" là kết quả gì? 
"draw on = to use information, experience, knowledge etc for a particular purpose" --> đề xuất dịch đơn giản là "dựa vào trực giác của họ"?

Reply:
Mình cũng thấy hơi khó dịch chỗ này nên mới thêm vào cụm "lấy kết quả". Vì nếu rút lại thành "bạn có thể dựa trên được trực giác của họ" thì câu hỏi bật ra là dựa vào để làm gì ?

Reply:
dựa trên trực giác để phân tích lỗi, như đề mục của bullet points. 
MÌnh thấy dịch "dựa trên trực giác" là đẹp rồi.